### ISSUE TITLE
Python client?

### ISSUE THREAD

**Issue by mdecuir** [Netflix/conductor#63]
Where do alternate client implementations fit in the roadmap for this project? I see a python_client branch but I do not see any conversation or activity about the branch for nearly a month, so it is not at all clear what the status of this is.

**Comment by v1r3n** [comment]
Feel free to experiment with the python_client from the branch.  The python client is under development and hence not released into main branch yet.  But overall, it is functional.

If you do end up trying it out, would love to hear your experience and any issues.

**Comment by mdecuir** [comment]
I will provide more detailed feedback as I explore deeper, but there are a few issues I have already encountered that feel like they are worth sharing immediately.

The code is written in python 2 syntax but nothing indicates as such, so I had to jump through some hoops to get it converted.

The branch is well behind development so it exposes some parameters and endpoints that don't seem to be supported with the current release version. The one specific case I have found so far is the api/queue/{taskType} endpoint which looks like it has been replaced with api/in_progress/{taskType}.

The ConductorWorker class does not look like it has any mechanisms to allow it to be cleanly stopped and can only be forcibly killed.

I will keep playing with this and let you know what I find.
